### ISSUE TITLE
[3.x]The engine "yarn" is incompatible with this module. Expected version ">= 3.6". Got "1.22.19"

### ISSUE THREAD

**Issue by onlyling** [DocSpring/craco-less#112]
> The engine "yarn" is incompatible with this module. Expected version ">= 3.6". Got "1.22.19"

How to use it in the lower version of yarn?

**Comment by abenhamdine** [comment]
I've got the same error while installing it : 

  error craco-less@3.0.0: The engine "yarn" is incompatible with this module. Expected version ">= 3.6". Got "1.22.19"
  error Found incompatible module.
  info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
  Error: Error: The process '/usr/local/bin/yarn' failed with exit code 1
  Error: The process '/usr/local/bin/yarn' failed with exit code 1

**Comment by onlyling** [comment]
> Is this while using it as an npm module in your project? I might have to remove the yarn engine plugin.

```bash
─onlyling@MBP14 ~/fruits-chain/web-template ‹next› 
╰─$ yarn add craco-less
yarn add v1.22.19
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error craco-less@3.0.0: The engine "yarn" is incompatible with this module. Expected version ">= 3.6". Got "1.22.19"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

```bash
node -v
v16.19.1

npm -v
8.19.3
```

**Comment by kamronbatman** [comment]
I believe I got it fixed with v3.0.1 publish. Let me know if you still have issues.

**Comment by onlyling** [comment]
> I believe I got it fixed with v3.0.1 publish. Let me know if you still have issues.

no more issues.

**Comment by abenhamdine** [comment]
thx a lot for this fix, it works fine now
